Foundation/NSOrderedSet.swift

```diff
@@ -37,38 +48,151 @@ public class NSOrderedSet : NSObject, NSCopying, NSMutableCopying, NSSecureCodin
     
     public required init?(coder aDecoder: NSCoder) { NSUnimplemented() }
     
-    public var count: Int { NSUnimplemented() }
-    public func objectAtIndex(idx: Int) -> AnyObject { NSUnimplemented() }
-    public func indexOfObject(object: AnyObject) -> Int { NSUnimplemented() }
-    public override init() { NSUnimplemented() }
-    public init(objects: UnsafePointer<AnyObject?>, count cnt: Int) { NSUnimplemented() }
+    public var count: Int {
+        return _entries.count
+    }
+
+    public func objectAtIndex(idx: Int) -> AnyObject {
+        return _entryAtIndex(idx).object
```

Comment:
I've gone ahead and done an implementation based on the original implementation - let me know if you prefer this approach over the linked-list version. Thanks!